packages/actions/src/makeSimpleActionCreator.js

```diff
@@ -1,13 +1,28 @@
-import { __, identity } from 'ramda';
-import { alwaysNull } from 'ramda-extension';
+import invariant from 'invariant';
+import { identity, always } from 'ramda';
 
 import makeActionCreator from './makeActionCreator';
 
+const alwaysUndefined = always(undefined);
```

Comment:
we're using `alwaysUndefined` in multiple places, let's create an `alwaysUndefined.js` file in the actions package. (no need to export it in index.js, of course)